geopandas/tools/_show_versions.py

```diff
@@ -48,8 +48,14 @@ def _get_C_info():
         geos_version = "{}.{}.{}".format(*shapely._buildcfg.geos_version)
         geos_dir = shapely._buildcfg.geos_library_path
     except Exception:
-        geos_version = None
-        geos_dir = None
+        try:
+            from shapely import geos_version_string
+
+            geos_version = geos_version_string
+            geos_dir = None
```

Comment:
With Shapely 2.0, shapely itself doesn't have a bunch of code to determine which GEOS library to load at runtime using ctypes, since it is now a normal compiled extension module. So we don't have a easy access to which GEOS library is actually being used. 

Do we still want to do some effort to actually get this information? (it might be useful sometimes when debugging environment / installation issues ..) 
Although if we want to do that, it will probably be some platform specific code .. (eg for linux I can do `lsof -p <pid> | awk '{print $9}' | grep 'libgeos\.so'` to get which GEOS lib is loaded in the python process)

ctypes has a `find_library` (https://docs.python.org/3/library/ctypes.html#finding-shared-libraries), which is also what shapely used before in some cases, but no idea how "correct" this is compared to what actually happens in practice (eg will it find the lib included in a wheel?)

Reply:
Well, it is useful information. I remember that some issues were already resolved by that and knowing the changes between GEOS versions (e.g. overlay refactoring). But I am not sure how easy would it be. What you describe doesn't sound like an easy solution.

Reply:
So it's certainly possible to do (eg with `lsof` on Linux, the same or `otool` on MacOS, although for Windows I am not sure there is a built-in command line (it seems you can easily see it through the Process Explorer UI)), but so potentially not for all platforms (certainly not the more esoteric ones), and some custom code. 
It has certainly been useful information in the past, but my hope is also that it will be less needed with shapely 2.0 being an extension module. But for example if we are happy to start with just showing it on maintstream Linux and MacOS, that shouldn't be too hard to add.